src/routes/contentRoutes.js

```diff
@@ -11,7 +11,8 @@ var BASE_URL = '/v1/content'
 
 module.exports = function (app) {
   app.route(BASE_URL + '/search')
-    .post(requestMiddleware.createAndValidateRequestBody, contentService.searchContentAPI)
+    .post(requestMiddleware.addChannelFilters, requestMiddleware.createAndValidateRequestBody,
```

Comment:
why is "addChannelFilters" middleware added before "createAndValidateRequestBody"? any specific reason? this applies to all APIs which have this middleware.